### ISSUE TITLE
Should rules have an assumption about aria-busy?

### ISSUE THREAD

**Issue by WilcoFiers** [auto-wcag/auto-wcag#2140]
This comes from an [ARIA review](https://github.com/w3c/aria/discussions/1850#discussioncomment-4309802)

We currently only have one rule that has an exception for aria-busy. I think for that rule it is a reasonable thing to have. I'm less sure about other rules, it seems unlikely that aria-busy is going to resolve issues such as invalid roles, or missing text alternatives. Even if not, should rules have an assumption that content is loaded?

**Comment by Jym77** [comment]
Decision:
add an Accessibility Support note to every rule using "Accessibility tree" as input aspect, stating that "`aria-busy` exists but isn't widely supported (JAWS only, currently, it seems), so the rule ignores it".
